src/caret_analyze/infra/lttng/ros2_tracing/processor.py

```diff
@@ -64,34 +67,36 @@ def __init__(
         ----------
         data : Ros2DataModel
             DataModel to be handles
+        addr_remapper : AddressRemapper
+            Addrees remapper to resolve address duplication.
         monotonic_to_system_time_offset : int | None
             Offset time to convert monotonic time to system time.
             This values should be valid number if a recording was done with runtime recording.
             None is given, if recording begins before launch of the application,
             to use time sampled from the trace point.
 
         """
-        from caret_analyze.infra.lttng.id_remapper import IDRemapper
-        self._context_handle_remapper = IDRemapper()
-        self._node_handle_remapper = IDRemapper()
-        self._rmw_node_handle_remapper = IDRemapper()
-        self._publisher_handle_remapper = IDRemapper()
-        self._rmw_publisher_handle_remapper = IDRemapper()
-        self._subscription_handle_remapper = IDRemapper()
-        self._rmw_subscription_handle_remapper = IDRemapper()
-        self._subscription_remapper = IDRemapper()
-        self._service_handle_remapper = IDRemapper()
-        self._rmw_service_handle_remapper = IDRemapper()
-        self._client_handle_remapper = IDRemapper()
-        self._rmw_client_handle_remapper = IDRemapper()
-        self._timer_handle_remapper = IDRemapper()
-        self._state_machine_remapper = IDRemapper()
-        self._executor_addr_remapper = IDRemapper()
-        self._entities_collector_addr_remapper = IDRemapper()
-        self._callback_group_addr_remapper = IDRemapper()
-        self._callback_remapper = IDRemapper()
-        self._buffer_remapper = IDRemapper()
-        self._ipb_remapper = IDRemapper()
+        # self._context_handle_remapper = IDRemapper()
+        # self._node_handle_remapper = IDRemapper()
+        # self._rmw_node_handle_remapper = IDRemapper()
+        # self._publisher_handle_remapper = IDRemapper()
+        # self._rmw_publisher_handle_remapper = IDRemapper()
+        # self._subscription_handle_remapper = IDRemapper()
+        # self._rmw_subscription_handle_remapper = IDRemapper()
+        # self._subscription_remapper = IDRemapper()
+        # self._service_handle_remapper = IDRemapper()
+        # self._rmw_service_handle_remapper = IDRemapper()
+        # self._client_handle_remapper = IDRemapper()
+        # self._rmw_client_handle_remapper = IDRemapper()
+        # self._timer_handle_remapper = IDRemapper()
+        # self._state_machine_remapper = IDRemapper()
+        # self._executor_addr_remapper = IDRemapper()
+        # self._entities_collector_addr_remapper = IDRemapper()
+        # self._callback_group_addr_remapper = IDRemapper()
+        # self._callback_remapper = IDRemapper()
+        # self._buffer_remapper = IDRemapper()
+        # self._ipb_remapper = IDRemapper()
```

Comment:
I suggest removing these comments.
```suggestion
```

Reply:
Fixed in [1d9ae6e](https://github.com/tier4/caret_analyze/pull/455/commits/1d9ae6e9cab36add6e776e71f2a5443db1d64d4f).